web5-test-vectors/vector-structure.json

```diff
@@ -0,0 +1,35 @@
+{
+    "description": "Describe the feature under test",
+    "numberOfTests": 3,
```

Comment:
Curious why this is needed, instead of counting the number of vectors in the test?

Reply:
+1

Reply:
i can remove, it was influenced by the wycheproof vectors

Reply:
I'm in favor of nixing.